source/io.c

```diff
@@ -122,24 +122,30 @@ PyObject *aws_py_io_client_tls_ctx_new(PyObject *self, PyObject *args) {
     struct aws_allocator *allocator = aws_crt_python_get_allocator();
 
     int min_tls_version = AWS_IO_TLS_VER_SYS_DEFAULTS;
-    const char *ca_file = NULL;
     const char *ca_path = NULL;
+    const char *ca_buffer = NULL;
+    int ca_buffer_len = 0;
```

Comment:
Py_ssize_t instead of int, otherwise you'll get some real gnar crashes later.

Reply:
Interesting, their docs literally say int:
```y# (read-only bytes-like object) [const char *, int]```

Reply:
Note that we [define `PY_SSIZE_T_CLEAN`](https://github.com/awslabs/aws-crt-python/blob/master/source/module.h#L22) (as per [Parsing arguments and building values](https://docs.python.org/2.7/c-api/arg.html)):

> Starting with Python 2.5 the type of the length argument can be controlled by defining the macro PY_SSIZE_T_CLEAN before including Python.h. If the macro is defined, length is a Py_ssize_t rather than an int.

Reply:
TIL